inst/extdata/gcam-usa/NREL_us_re_capacity_factors.csv

```diff
@@ -0,0 +1,56 @@
+# File: NREL_us_re_capacity_factors.csv,,,,,,
+# Title: Capacity factors by state by renewable technology,,,,,,
+# Units: Unitless,,,,,,
+State,Urban_Utility_scale_PV,Rural_Utility_scale_PV,Rooftop_PV,CSP,Onshore_Wind,Offshore_Wind
```

Comment:
We can either hold this PR until you find out more, or merge and open an issue (so we don't forget about this). Your choice.